src/about.njk

```diff
@@ -79,17 +79,17 @@
         <p class="text-muted">Knoxville leader</p>
         <ul class="list-inline social-buttons">
           <li class="list-inline-item">
-            <a href="https://github.com/stefanhk31" target="_blank" class="shadow-sm">
+            <a href="https://github.com/stefanhk31" target="_blank" class="shadow-sm" aria-label="Navigation link to Stefan Hodges-Kluck GitHub Page">
               <i class="fab fa-github"></i>
             </a>
           </li>
           <li class="list-inline-item">
-            <a href="https://www.stefanhodgeskluck.com" target="_blank" class="shadow-sm">
+            <a href="https://www.stefanhodgeskluck.com" target="_blank" class="shadow-sm" aria-label="Navigation link to Stefan Hodges-Kluck personal web page">
               <i class="fas fa-globe"></i>
             </a>
           </li>
           <li class="list-inline-item">
-            <a href="https://www.linkedin.com/in/stefanhk31/" target="_blank" class="shadow-sm">
+            <a href="https://www.linkedin.com/in/stefanhk31/" target="_blank" class="shadow-sm" aria-label="Navigation link to Stefan Hodges-Kluck LinkedIn page">
```

Comment:
Screen readers will announce this as a link, so a better aria-label would be "Stefan Hodges-Kluck's LinkedIn page"